.gitignore

```diff
@@ -1,3 +1,6 @@
 ui.log
 .idea/
 __pycache__/
+_android_web/
+**/settings.json*
```

Comment:
It will ignore the _nodejs/settings.json too. Not sure that it's correct.

Reply:
@dubrsl Thanks for pointing out, fixed.